packages/design-system/src/styles/components/_Table.scss

```diff
@@ -30,26 +30,32 @@ $table-stacked-header-max-width: 160px !default;
   th {
```

Comment:
Can we move [these styles](https://github.com/CMSgov/design-system/pull/853/files#diff-2b709213c3173751c76a6658e5db4facf82984bc78f95a2598faf72564b4ebcfL77-L78) under this block? This way all `TableCell` styles are in the same block.

Reply:
Which styles to move to the `th` block on line 30? The link for `these styles` doesn't seem to work.